fetch.bs

```diff
@@ -2864,6 +2914,33 @@ run these steps:
  <li><p>Return <b>allowed</b>.
 </ol>
 
+<h3 dfn export lt="is a cors requests header size over the limit" id=is-a-cors-requests-header-size-over-the-limit>
+Is a CORS request's header size over the limit?</h3>
```

Comment:
We're not necessarily talking about a CORS request here, are we? I'd prefer not having the algorithm name in the heading if we can avoid it.